src/Tribe/Tickets_Handler.php

```diff
@@ -57,6 +57,7 @@ public function __construct() {
 
 		add_action( 'admin_menu', array( $this, 'attendees_page_register' ) );
 		add_filter( 'post_row_actions', array( $this, 'attendees_row_action' ) );
+		add_filter( 'page_row_actions', array( $this, 'attendees_row_action' ) );
```

Comment:
From what I read CPT's will use post_row_actions only pages needed the other action.